plotter/db.py

```diff
@@ -166,20 +166,28 @@ def close_db():
     env.close()
 atexit.register(close_db)
 
+DB_CLASSES = []
```

Comment:
this is a new list of classes that inherit from `Resource`, and have BerkeleyDB databases